opentelemetry-sdk/benchmarks/trace/test_benchmark__init__.py

```diff
@@ -0,0 +1,51 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
We could just stick these under `opentelemetry-sdk/tests/benchmarks` to avoid having to create new tox/CI targets for them. Thoughts?

Reply:
That's actually what I had originally, but I observed the following:
- I didn't like how I had to wait through a bunch of tests (235 tests for `opentelemetry-sdk`) just to see benchmarks at the end. 
- Also, the `benchmark` CI flow is simpler and doesn't need the same items as the `build` (test) workflow so I liked how the GitHub actions had a separate benchmark section where you could go and read if your code failed benchmarks (even though it pass unit tests)
- I could see there being _additional_ steps to specifically the benchmark work (like write a file with the performance results, publish the results, compare the results, fail the PR if the perfomance went down, etc...) so that's the usefulness of having a different CI for it

Reply:
I went ahead with your ask and even printed the results to the GitHub workflow run (it works locally too!) with all tests under `opentelemetry-sdk/tests/performance/benchmarks` (because I want to add `opentelemetry-sdk/tests/performance/resource-usage` soon.